mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/FullTextField.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.mapping.definition.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.hibernate.search.engine.backend.document.model.dsl.Store;
+
+/**
+ * A full-text field in the full text index, potentially holding multiple tokens (words) of text.
+ * <p>
+ * Note that this annotation only creates tokenized (multi-word) text fields.
+ * As a result:
+ * <ul>
+ *     <li>You must assign an analyzer when using this annotation</li>
+ *     <li>This annotation does not allow to make the field sortable (analyzed fields cannot be sorted on)</li>
+ * </ul>
+ * <p>
+ * If you don't want the field to be analyzed, or want it to be sortable,
+ * use the {@link GenericField} annotation instead.
```

Comment:
Wouldn't we recommend `KeywordField`?

Reply:
Right. I clarified the documentation to also mention we expect the field to be of type String, and that KeywordField should be used in this case if the field is a String.